app/views/deferred/confirmations/_canned_responses.html.haml

```diff
@@ -39,6 +39,17 @@
           %p.form-hint eg Last 8 digits of VO number or "none" for remand
           %span.validation-message= f.object.errors[:vo_number].first if f.object.errors.include?(:vo_number)
         = text_field_tag 'confirmation[vo_number]', nil, :maxlength => 8
+      
+      .group
+        #closed_visit.field
+          %label.block-label(for='confirmation_closed_visit')
+            = check_box_tag 'confirmation[closed_visit]', 1, f.object.closed_visit.present?, { :'data-conditional-el' => 'details_closed_visit', :class => 'js-Conditional' }
+            This is a closed visit
+
+        #details_closed_visit
+          %h3 What we’ll tell the visitor
+          %p
+            %em.email This is a closed visit.
```

Comment:
Does this need to exactly reflect the text we show the visitor, or is this enough?